### PR TITLE
Cleaned up peerDependencies in bits

### PR DESCRIPTION
**Pull request by lukasholcik**

### CODE REVIEW

File: packages/bits/package.json [hunk 1]
```diff
@@ -89,25 +89,17 @@
   },
   "analyze": false,
   "peerDependencies": {
-    "@angular-devkit/schematics": "^11.1.4",
```
Comment (lukasholcik) [inline packages/bits/package.json:92]: I haven't seen this library in any of the angular libraries' peerDependencies. Schematics work without it.

File: packages/bits/package.json [hunk 2]
```diff
@@ -89,25 +89,17 @@
...
     "@angular/animations": "^11.1.2",
     "@angular/cdk": "^11.1.1",
-    "@angular/common": "^11.1.2",
-    "@angular/compiler": "^11.1.2",
     "@angular/core": "^11.1.2",
     "@angular/forms": "^11.1.2",
     "@angular/localize": "^11.1.2",
     "@angular/platform-browser": "^11.1.2",
-    "@angular/platform-browser-dynamic": "^11.1.2",
```
Comment (lukasholcik) [inline packages/bits/package.json:101]: platform-browser-dynamic is only used in tests.

File: packages/bits/package.json [hunk 3]
```diff
@@ -89,25 +89,17 @@
   },
   "analyze": false,
   "peerDependencies": {
-    "@angular-devkit/schematics": "^11.1.4",
     "@angular/animations": "^11.1.2",
     "@angular/cdk": "^11.1.1",
-    "@angular/common": "^11.1.2",
```
Comment (lukasholcik) [inline packages/bits/package.json:95]: common and compiler are a natural dependency of `forms` and are included in all Angular apps.

File: packages/bits/package.json [hunk 4]
```diff
@@ -89,25 +89,17 @@
...
-    "@angular/compiler": "^11.1.2",
     "@angular/core": "^11.1.2",
     "@angular/forms": "^11.1.2",
     "@angular/localize": "^11.1.2",
     "@angular/platform-browser": "^11.1.2",
-    "@angular/platform-browser-dynamic": "^11.1.2",
     "@angular/router": "^11.1.2",
-    "@schematics/angular": "^11.1.4",
```
Comment (lukasholcik) [inline packages/bits/package.json:103]: schematics/angular is not necessary as peer.

File: packages/bits/package.json [hunk 5]
```diff
@@ -89,25 +89,17 @@
...
-    "core-js": "^2.6.9",
```
Comment (lukasholcik) [inline packages/bits/package.json:104]: core-js is apparently used only for backward compatibility with IE.

File: packages/bits/package.json [hunk 6]
```diff
@@ -89,25 +89,17 @@
...
     "highlight.js": "^10.6.0",
     "lodash": "^4.17.21",
     "moment": "^2.25.3",
-    "moment-timezone": "^0.5.27",
-    "rxjs": "^6.5.3",
```
Comment (lukasholcik) [inline packages/bits/package.json:109]: rxjs and zone.js are required by angular/core, so not necessary to mention them here.

**Review by Blakesalot — APPROVED** [review]

**Review by madelyneh — APPROVED** [review]

**Pull request merged**